src/raftstore/store/metrics.rs

```diff
@@ -136,6 +137,68 @@ lazy_static! {
             &["cf"]
         ).unwrap();
 
+    pub static ref STORE_ENGINE_CACHE_EFFICIENCY_VEC: GaugeVec =
```

Comment:
any update?
I suggest using a `rocksdb_metrics.rs` for all rocksdb metrics. 
Btw, I also think we should move the rocksdb metric get from `store.rs` to another file.